docs/users/api/art.rst

```diff
@@ -0,0 +1,28 @@
+Art
+===
+
+ListenBrainz has a (cover) art infrastructure that creates new cover art from a user's statstics or
+a user's instructions on how to composite a cover art grid.
+
+All art endpoints have this root URL for our current production site.
+
```

Comment:
It might be worth to add a note for the usage of the SVG the endpoints return:

As these endpoints return SVGs rather than images, you must embed them in an html `<object data="covert_art_url" type="image/svg+xml">` element* rather than an `<img src="covert_art_url">` element.
Otherwise external resources such as cover art images  and fonts will not be loaded and the result will be useless.

* See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/object for reference